tests/functional/test_tracing.py

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+from time import sleep
+import pytest
+
+from core.test_run import TestRun
+from utils.iotrace import IotracePlugin
+from storage_devices.disk import DiskType, DiskTypeLowerThan, DiskTypeSet
+
+
+def test_iotracer_multiple_instances_same_disks():
+    """
+        title: Check if it’s possible to run concurrent tracing processes.
+        description: |
+          Try to run concurrent io-tracers for the same disks.
+        pass_criteria:
+          - No system crash.
+          - It’s impossible to run io-tracer multiple times for the same disks.
+    """
+    with TestRun.step("Run the first io-tracer instance"):
+        iotracer1 = IotracePlugin(TestRun.plugins['iotrace'].repo_dir,
```

Comment:
Please consider test according to new iotrace plugin constructor 
I'd use first plugin as usual, second tracing I'd execute by calling command directly.

Reply:
I've updated initialization of objects, but if I'd run command directly, I won't check status of the second expected-to-fail iotrace instance.